traffic_ops/client

```diff
@@ -1 +1 @@
-v3-client
\ No newline at end of file
+v4-client
```

Comment:
We can actually just remove this symlink now. It's use was deprecated in the last release.

Reply:
Wouldn't we need to keep it until API v3 is no longer used/ATS v5.0 is unsupported?

Reply:
No, use of the un-versioned `client` import was deprecated in the last major release specifically so we could remove it now. Each version-specific library remains unbroken, you just have to use the correct path. Versioned paths have existed since 4.0 iirc, so people have had plenty of time to stop using this crutch.